src/utils/keyRotation.ts

```diff
@@ -12,18 +12,24 @@ export function shouldRotateKey(date: Date, env: Bindings): boolean {
 }
 
 function matchCronTime(cronString: string, date: Date): boolean {
+	// Set seconds and milliseconds to 0 to round to the nearest minute
```

Comment:
this is a truncation, not a round

```suggestion
	// Set seconds and milliseconds to 0 to truncate to the minute
```